soccer/src/soccer/strategy/coach/coach_node.cpp

```diff
@@ -76,6 +81,10 @@ void CoachNode::ball_sense_callback(const rj_msgs::msg::RobotStatus::SharedPtr m
 }
 
 void CoachNode::check_for_play_state_change() {
+    assign_positions();
+
+    // TODO: assign positions on play state changes correctly
```

Comment:
@p-nayak11 can you add a card for  this?
```suggestion
    // TODO(p-nayak): assign positions on play state changes correctly
```

Reply:
This has been overridden by the comment that follows. Resolved.